crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -288,7 +288,19 @@ pub enum AdminRequest {
         cell_id: Option<CellId>,
     },
 
-    /// Insert [`Record`]s into the source chain of the [`CellId`].
+    /// "Graft" [`Record`]s onto the source chain of the specified [`CellId`].
+    ///
+    /// The records must form a valid chain segment (ascending sequence numbers,
+    /// and valid `prev_action` references). If the first record contains a `prev_action`
+    /// which matches the existing records, then the new records will be "grafted" onto
+    /// the existing chain at that point, and any other records above that one which do
```

Comment:
```suggestion
    /// the existing chain at that point, and other existing records following that point which do
```
I understand that "above" refers to the existing records in ascending order. I find it clearer to the "uninitiated" like this ;-)